deploy/infra/modules/aci.bicep

```diff
@@ -0,0 +1,114 @@
+@description('Name for the container group')
+param name string
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Container image to deploy. Should be of the form repoName/imagename:tag for images stored in public Docker Hub, or a fully qualified URI for other registries. Images from private registries require additional registry credentials.')
+param image string = 'mcr.microsoft.com/oss/mirror/docker.io/library/postgres:12.9-bullseye'
+
+@description('Port to open on the container and the public IP address.')
+param port int = 80
+
+@description('The number of CPU cores to allocate to the container.')
+param cpuCores int = 1
+
+@description('The amount of memory to allocate to the container in gigabytes.')
+param memoryInGb int = 4
+
+param userManagedIdentityId string
+
+param userManagedIdentityPrincipalId string
+
+param assignRoleToUserManagedIdentity string = 'Owner'
+
+@description('The behavior of Azure runtime if container has stopped.')
+@allowed([
+  'Always'
+  'Never'
+  'OnFailure'
+])
+param restartPolicy string = 'Never'
+
+param server string
+
+param db string = 'postgres'
+
+param username string 
+
+@secure()
+param dbPassword string
+
+param dbPort int = 5432
+
+var commandString = 'apt update -y && apt install curl -y && curl -o root.crt https://cacerts.digicert.com/BaltimoreCyberTrustRoot.crt.pem && curl -o CVModelSQLScript.sql https://raw.githubusercontent.com/Azure/Azure-Orbital-Analytics-Samples/main/deploy/CVModelSQLScript.sql && psql --set=sslmode=require --set=sslrootcert=root.crt -h ${server} -p ${dbPort} -U ${username} -W -d ${db} -f CVModelSQLScript.sql'
```

Comment:
until this code is merged, this will not work? https://raw.githubusercontent.com/Azure/Azure-Orbital-Analytics-Samples/main/deploy/CVModelSQLScript.sql return 404.

Reply:
any reason we want to run this command in aci? If decision is made, I think this commandString should be a parameter so that we would make aci.bicep module generic and reusable for other contexts.